epochtalk/core-pg
# Board bans

Test Procedure: 

1) psql to epochtalk db, then run "DELETE FROM roles;"
2) Run "npm run db-migrate"
3) See frontend test proceedure
